lib/widgets/tb_elevated_button.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:tech_byte/utils/colors.dart';
+import 'package:tech_byte/utils/constants.dart';
+
+enum TBButtonType { primary, secondary }
+
+class TBButton extends StatelessWidget {
+  final void Function() onPressed;
+  final String text;
+  final bool enabled;
+  final TBButtonType type;
+  final bool isLoading;
+
+  const TBButton({
+    super.key,
+    required this.onPressed,
```

Comment:
onPressed shouldn't be required, it is how we determine if it is enabled or disabled.